### PR TITLE
Add a function that provides fees for the transactions

### PR DESCRIPTION
**Pull request by MichaelKim20**
Fees increase with the size of the transaction.
When an endpoint is added to Stoa, It will be changed to use the endpoint of Stoa to obtain transaction fees.

Relates to https://github.com/bpfkorea/stoa/issues/274

### CODE REVIEW

**Comment by MichaelKim20** [comment]
The code was fixed because 0.01BOA is 100_000.
And I added a test code that specifies the accurate fee using the transaction build.
This process is a bit tricky because we can only know the size of the transaction after it is created.

File: tests/BOAClient.test.ts [hunk 1]
```diff
@@ -261,6 +261,41 @@ export class TestStoa {
                 res.status(200).send("10");
             });
 
+        // http://localhost/transaction/fees
+        this.app.get("/transaction/fees/:size",
```
Comment (TrustHenry) [inline tests/BOAClient.test.ts:265]:
```suggestion
        this.app.get("/transaction/fees/:tx_size",
```

File: tests/BOAClient.test.ts [hunk 2]
```diff
@@ -968,4 +1003,185 @@ describe('BOA Client', () => {
...
+
+        fees = await boa_client.getTransactionFee(1_000);
+        assert.strictEqual(fees.medium, "200000");
+        assert.strictEqual(fees.low, "180000");
+        assert.strictEqual(fees.high, "220000");
+
+        fees = await boa_client.getTransactionFee(100_000);
+        assert.strictEqual(fees.medium, "20000000");
+        assert.strictEqual(fees.low, "18000000");
+        assert.strictEqual(fees.high, "22000000");
```
Comment (TrustHenry) [inline tests/BOAClient.test.ts:1033]: 100Kbyte Medium Fee  = 1.8 BOA  :)

Comment (MichaelKim20) [reply tests/BOAClient.test.ts:1033]: > 100Kbyte Medium Fee = 2 BOA :)

500 : 0.01 BOA = 100,000 = x 
x = (0.01 * 100000) / 500 = 2

Comment (MichaelKim20) [reply tests/BOAClient.test.ts:1033]: If there are 2,000 outputs, it's about 80K bytes.

File: tests/BOAClient.test.ts (hunk 1 above)
Comment (MichaelKim20) [reply tests/BOAClient.test.ts]: I fixed it

**Review by TrustHenry — APPROVED** [review]
LGTM

File: src/modules/data/Transaction.ts [hunk 3]
```diff
@@ -140,4 +140,14 @@ export class Transaction
 
         return bytes_length;
     }
+
+    /**
+     * Returns the estimated transaction size.
+     */
+    public static getEstimatedNumberOfBytes (num_input: number, num_output: number, num_bytes_payload: number): number
```
Comment (TrustHenry) [inline src/modules/data/Transaction.ts:147]: A parameter document is required.

Comment (MichaelKim20) [reply src/modules/data/Transaction.ts]: I added the documentation of the params

**Pull request merged**